Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.lean

```diff
@@ -321,20 +321,20 @@ theorem sin_int_mul_two_pi_sub (x : ℝ) (n : ℤ) : sin (n * (2 * π) - x) = -s
   sin_neg x ▸ sin_periodic.int_mul_sub_eq n
 #align real.sin_int_mul_two_pi_sub Real.sin_int_mul_two_pi_sub
 
-theorem sin_add_int_mul_pi (x : ℝ) (n : ℤ) : sin (x + n * π) = n.negOnePow * sin x :=
-  sin_antiperiodic.add_int_mul_eq n
+theorem sin_add_int_mul_pi (x : ℝ) (n : ℤ) : sin (x + n * π) = (-1) ^ n * sin x :=
+  @Int.coe_negOnePow ℝ _ n ▸ sin_antiperiodic.add_int_mul_eq n
```

Comment:
```suggestion
  Int.coe_negOnePow n ▸ sin_antiperiodic.add_int_mul_eq n
```
If this doesn't work, make sure to turn the `K` argument explicit in `Int.coe_negOnePow`